server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -582,6 +645,12 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
                     break;
                 }
 
+                case ControlMessageType.LambdaStartResult: {
+                    const controlContents = controlMessage.contents as ILambdaStartControlMessageContents;
+                    this.scribeStartSuccess = controlContents.success;
```

Comment:
I would like it if `ILambdaStartControlMessageContents` was a bit more generic. 
For example, it could include a `lambdaName: string` and scribe can set it to `scribe`. Then deli can keep track of `startedLambdas: string[]`.  So `if (this.scribeStartSuccess) {` would be replaced by `if (this.startedLambdas.includes("string")) {`

That would make it easy for us to extend this capability to more lambdas in the future